google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/GoogleCloudApiSelectorPanel.java

```diff
@@ -224,6 +249,27 @@ private void updateManagementUI() {
     detailsPanel.setManagementUIEnabled(addLibrary && projectSelector.getSelectedProject() != null);
   }
 
+  /**
+   * Populates the BOM {@link JComboBox} with the fetched versions. If there are no versions
+   * returned, then the BOM UX is hidden.
+   */
+  // TODO (eshaul): make async with loader icons
+  private void populateBomVersions() {
+    List<Version> bomVersions =
+        CloudApiMavenService.getInstance().getBomVersions(NUM_BOM_VERSIONS_TO_SHOW);
+
+    if (bomVersions.isEmpty()) {
+      hideBomUI();
+    } else {
+      bomVersions.forEach(bomComboBox::addItem);
```

Comment:
Will this always show proper version display name, i.e. `toString()` method is stable for `Version` and no separate renderer is needed?

Reply:
yes, looks like the toString method itself is smart enough to render the version properly